german/Hilfeschrei/Vertrauen/insMaul/ins_maul

```diff
@@ -0,0 +1,2 @@
+Zum Glück haben Sie eine kleine Taschenlampe am Schlüsselbund und beginnen
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.